napari/layers/_shapes_layer/shapes/ellipse.py

```diff
@@ -104,3 +105,44 @@ def to_mask(self, mask_shape=None):
         mask = poly_to_mask(mask_shape, self._face_vertices)
 
         return mask
+
+    def to_xml(self):
+        """Generates an xml element that defintes the shape according to the
+        svg specification.
+
+        Returns
+        ----------
+        element : xml.etree.ElementTree.Element
+            xml element specifying the shape according to svg.
+        """
+        props = self.svg_props
+
+        offset = self.data[1] - self.data[0]
+        angle = -np.arctan2(offset[0], -offset[1])
+        if not angle == 0:
+            # if shape has been rotated, shift to origin
+            cen = self.data.mean(axis=0)
+            coords = self.data - cen
+
+            # rotate back to axis aligned
+            c, s = np.cos(angle), np.sin(-angle)
+            rotation = np.array([[c, s], [-s, c]])
+            coords = coords @ rotation.T
+
+            # shift back to center
+            coords = coords + cen
+
+            # define rotation around center
+            transform = f'rotate({np.degrees(-angle)} {cen[0]} {cen[1]})'
+            props['transform'] = transform
+        else:
+            coords = self.data
+
+        cx = str(cen[0])
+        cy = str(cen[1])
+        size = abs(coords[2] - coords[0])
+        rx = str(size[0]/2)
```

Comment:
you should include whitespace around operators:
```suggestion
        rx = str(size[0] / 2)
```

Reply:
fixed